jobs/JGLOBAL_ATMOS_POST

```diff
@@ -1,46 +1,51 @@
 #! /usr/bin/env bash
 
-source "$HOMEgfs/ush/preamble.sh"
+# shellcheck source=ush/preamble.sh
+source "${HOMEgfs:?}/ush/preamble.sh"
 
 export RUN_ENVIR=${RUN_ENVIR:-"nco"}
 
 #############################
 # Source relevant config files
 #############################
 configs="base post"
-export EXPDIR=${EXPDIR:-$HOMEgfs/parm/config}
-config_path=${EXPDIR:-$PACKAGEROOT/gfs.${gfs_ver}/parm/config}
-for config in $configs; do
-    . $config_path/config.$config
+export EXPDIR=${EXPDIR:-${HOMEgfs}/parm/config}
+config_path=${EXPDIR:-${PACKAGEROOT}/gfs.${gfs_ver}/parm/config}
+for config in ${configs}; do
+    # shellcheck source-path=parm/config
+    . "${config_path}/config.${config}"
     status=$?
-    [[ $status -ne 0 ]] && exit $status
+    (( status != 0 )) && exit "${status}"
 done
 
 
 ##########################################
 # Source machine runtime environment
 ##########################################
-. $HOMEgfs/env/${machine}.env post
+# shellcheck source-path=env
```

Comment:
are we adding this now to every file?
Can this be set in higher level env?

Reply:
No, but we may be able to do without it. I was hoping to get fewer other complaints if shellcheck can source the right files, and I'm not sure it is even helping (especially this one, which I don't think is followed).